Composer/packages/client/src/utils/pageLinks.ts

```diff
@@ -42,17 +42,17 @@ export const topLinks = (
     {
       to: linkBase + `dialogs/${openedDialogId}`,
```

Comment:
Can you update all the `to` links to use string interpolation instead of using the addition operator? 
```suggestion
      to: `${linkBase}dialogs/${openedDialogId}`,
```